shell_integration/windows/OCContextMenu/OCClientInterface.cpp

```diff
@@ -45,7 +45,8 @@ OCClientInterface::ContextMenuInfo OCClientInterface::FetchInfo()
     if (!socket.Connect(pipename)) {
         return {};
     }
-    socket.SendMsg(L"GET_STRINGS\n");
+    socket.SendMsg(L"GET_STRINGS:CONTEXT_MENU_TITLE\n");
```

Comment:
I don't think you can request a particular string only right now.

Reply:
See the change in SocketApi::command_GET_STRINGS